tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 distribute = false
-envlist = {py27}-{test}
+envlist = {py27}-{test}, {py3}-{test}
```

Comment:
`{py27, py3}-test `
instead ?

Can you please update the `make clean`
So as the py3 xml output is cleaned up after tox.
`pytests_py*-test.xml`